### ISSUE TITLE
Question about the packages path prefix

### ISSUE THREAD

**Issue by a-carvallo** [loic-sharma/BaGet#329]
Hello,

When configuring the storage path I thought the packages would be in the specified path, but they are actually one directory deeper, in "packages".

It looks like it's due to the following line (and its uses):
https://github.com/loic-sharma/BaGet/blob/e57d120349b4a5fe34dde7952daf4a5dd8e68a7a/src/BaGet.Core/Storage/PackageStorageService.cs#L13

Is there a specific reason/need for having this prefix ?

Could it be possible to remove it, or have a config option to disable/change it ? (changing it seems a bit useless since you can already define the storage path).

**Comment by loic-sharma** [comment]
> Is there a specific reason/need for having this prefix ?

Yes, this prefix is used to differentiate between packages and symbols (see [the symbol storage service](https://github.com/loic-sharma/BaGet/blob/master/src/BaGet.Core/Storage/SymbolStorageService.cs#L11))

> Could it be possible to remove it, or have a config option to disable/change it ? (changing it seems a bit useless since you can already define the storage path).

Why would you like to change this prefix?

**Comment by a-carvallo** [comment]
> > Is there a specific reason/need for having this prefix ?
> 
> Yes, this prefix is used to differentiate between packages and symbols (see [the symbol storage service](https://github.com/loic-sharma/BaGet/blob/master/src/BaGet.Core/Storage/SymbolStorageService.cs#L11))

I see, I didn't really have in mind the whole symbols part, when you don't intend to use symbols it's a bit unexpected.

> > Could it be possible to remove it, or have a config option to disable/change it ? (changing it seems a bit useless since you can already define the storage path).
> 
> Why would you like to change this prefix?

I just wanted my packages to be in the specified folder, and not in another subfolder.
